data/tools/keploy.yml

```
@@ -0,0 +1,21 @@
+name: keploy
+categories:
+  - linter
+tags:
+  - keploy
+  - go
```

```suggestion
  - csharp
  - go
  - javascript
  - java
  - python
  - rust
```

According to [the docs](https://keploy.io/docs/).